modules/autoscaling.md

```diff
@@ -96,12 +96,40 @@ Cumulus's autoscaling module has the following usage:
 cumulus autoscaling [diff|help|list|migrate|sync] <asset>
 {% endhighlight %}
 
-Autoscaling groups can be diffed, listed, and synced (migrate is covered in the [following section](#migration)). These three actions do the following:
+Autoscaling groups can be diffed, listed, and synced (migrate is covered in [migration](#migration)). These three actions do the following:
 
 * `diff` - Shows the differences between the local definition and the AWS autoscaling configuration. If `<asset>` is specified, Cumulus will diff only the autoscaling group with that name.
 * `list` - Lists the names of all the autoscaling groups defined in local configuration
 * `sync` - Syncs local configuration with AWS. If `<asset>` is specified, Cumulus will sync only the autoscaling group with that name.
 
+
+### Syncing Size
+
+Because autoscaling groups can change in size in response to alarms or scheduled actions, the size (min, max, and desired instances) of an autoscaling group is synced a little differently than the other configuration options. If the local configuration has any scheduled actions configured, we will set the min and max size of the autoscaling group to match the last scheduled action that would have ran. Desired is never update unless it is not in the min/max bounds. This behavior can be overridden by passing the `--autoscaling-force-size` argument with the sync command, which will use the configured size on the autoscaling group instead of the most recent scheduled action.
+
+For example, if we have an autoscaling group configured as follows:
+
+{% highlight json %}
+{
+  ...
+  "scheduled": [
+    {
+      "name": "Scale up at 9am UTC",
+      "recurrence": "0 9 * * *",
+      "min": 9
+    }
+  ],
+  "size": {
+    "min": 3,
+    "max": 21,
+    "desired": 4
+  }
+  ...
+}
+{% endhighlight %}
+
+and the current time is 10am UTC, we will use a `min` value of 9, a `max` value of 21, and a `desired` value of 9 when comparing the local configuration to AWS configuration. Notice that if the `desired` value in AWS is outside of the new range, it will be updated to be within the range by raising the value to the `min` or lowering it to the `max`. If using the `--autoscaling-force-size` argument when diffing or syncing, the values of 3, 21, and 4 will be used for `min`, `max`, and `desired` respectively.
```

Comment:
I don't love having the sentence continue after the code, since reading the code kinda breaks sentence continuity. I would probably do the following:

Suppose we have the following autoscaling group configuration.
<code>
If the current time is 10am UTC...